test/ejabberd_tests/test.config

```diff
@@ -31,6 +31,10 @@
                  {domain, <<"localhost">>},
                  {vars, "node2_vars.config"},
                  {cluster, mim}]},
+         {mim3, [{node, ejabberd3@localhost},
```

Comment:
Yes, some tests may fail because of that :) so that's work for separate PR I think.